civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -83,3 +81,13 @@ $row-margin-bottom: 15px;
   margin-top: 7px;
   display: inline-block;
 }
+
+.chr_leave-request-modal__disabled {
+  opacity: 0.5;
+  pointer-events: none;
+
+  > * {
+    opacity: 0.5;
```

Comment:
Is this necessary? both `opacity` and `pointer-events` should work for all the children even if you dont mention it.